src/configure/configure.ts

```diff
@@ -96,7 +97,7 @@ class Orchestrator {
             await pipelineConfigurer.createPreRequisites(this.inputs, !!this.azureResourceClient ? this.azureResourceClient : new AppServiceClient(this.inputs.azureSession.credentials, this.inputs.azureSession.environment, this.inputs.azureSession.tenantId, this.inputs.subscriptionId));
 
             telemetryHelper.setCurrentStep('CreateAssets');
-            await new AssetHandler().createAssets(this.inputs.pipelineConfiguration.template.assets, this.inputs, (name: string, type: TemplateAssetType, data: any, inputs: WizardInputs) => { return pipelineConfigurer.createAsset(name, type, data, inputs); });
+            await new AssetHandler().createAssets(this.inputs.pipelineConfiguration.template.assets, this.inputs, (name: string, templateType: TemplateAssetType, data: any, inputs: WizardInputs) => { return pipelineConfigurer.createAsset(name, templateType, data, inputs); });
```

Comment:
type here signifies the asset type, hence `type` was a better name

Reply:
I reverted this change, it is giving ts-lint error that "type" is a reserved word can't use it as a variable name. That's why I changed it.

Reply:
in that case change it to `assetType`

Reply:
left it to type, will fix these issues when we will fix the ts-lint errors.